.ci.yaml

```diff
@@ -4915,3 +4915,17 @@ targets:
       validation: docs_deploy
       validation_name: Docs_deploy
       firebase_project: docs-flutter-dev
+
+  - name: Windows windows_app_template_test
+    recipe: devicelab/devicelab_drone
```

Comment:
Does this need to be a devicelab test? These are "expensive" tests that run on limited physical devices. Can we run this test on whatever runs our unit tests?

Reply:
Would that be `flutter/flutter_drone`? I believe that and `devicelab` run integration tests currently.

Reply:
How long does this take? If it's <=1 minute, we can add it to https://github.com/flutter/flutter/blob/master/dev/bots/analyze.dart

Reply:
> Would that be `flutter/flutter_drone`? I believe that and `devicelab` run integration tests currently.

The issue is not so much concurrency, but the fact that we have a limited number of physical devicelab bots, and scheduling an additional build per commit will slightly increase queue time for all other builds needing that bot.

Reply:
I will try adding it to `analyze.dart` then. It is a quick task.

Reply:
@christopherfujino Do you know the file system set up that the analyze script is run on? I am trying to add this test to analyze as you suggested, and the requested file [is not found](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8791601670423280513/+/u/Analyze/analyze/test_stderr), though it is when I run locally. Does the logic of getting the file paths in https://github.com/flutter/flutter/blob/4564aa854c25a9b395a8f324f8f506c72ad6b6e3/dev/bots/analyze.dart#L1890-L1903 seem mistaken to you?

Reply:
See https://github.com/flutter/flutter/pull/118646/files#r1080535963